src/unsplash.js

```diff
@@ -95,6 +95,11 @@ export function getImagesURLsForItems (items, { searchTerm, photoId }) {
           if (json.errors) {
             return Promise.reject(json.errors[0])
           }
+          if (!Array.isArray(json)) {
```

Comment:
Ah good catch, I missed it.

But I think there is a more efficient fix: instead of doing the whole `Promise.all` thing, we can just skip it completely when there is a photoId. That way we only do 1 API call.
Does that make sense?

Reply:
Good idea! Added a new commit that skips it.

However, DataProvider still fetches the same image multiple times for each frame size. We might want to fix that as well.

Reply:
That's a good point. Could you open a follow up issue? It doesn't need to be fixed here